Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -615,7 +656,6 @@
           DefaultPrototype: Beaker
           ExamineName: Glass Beaker
     - type: ReverseEngineering # Nyano
-      difficulty: 2
```

Comment:
Why is a heater harder to research than a cloning pod?

Reply:
that was an acident

Reply:
I think I fixed it in a latter commit, but it's back at four now... the cloner that is.